test.md

```diff
@@ -615,38 +615,50 @@ A standard note is formatted like this:
 > nodes.
 ```
 
-It renders like this with a colored sidebar and icon:
+A note renders as follows:
 
-![note admonition example](/images/note-admonition-example.png)
```

Comment:
Also see my other comment; when viewed on GitHub, the example won't show what it will look like in the docs (probably the image was used for that reason)